doc/api.apib

```diff
@@ -20,7 +20,7 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 
 # Group Workflows
 
-## Workflow [/v1/workflows/{component}/{workflow_id}]
```

Comment:
(Just playing a devil's advocate. I'd like outdated stuff to disappear immediately.)